app/assets/javascripts/foundation-datetimepicker.js

```diff
@@ -312,6 +312,30 @@
             });
         },
 
+        pickOnlyDate: function (e) {
+          this.minView = 2;
+          this.viewSelect = 2;
+          this.setFormat('yyyy-mm-dd');
+        },
+
+        pickDateAndTime: function (e) {
+          this.minView = 0;
+          this.viewSelect = 0;
+          this.setFormat('yyyy-mm-dd hh:ii');
```

Comment:
is the format the same for all languages? maybe you need to use these variables?
https://github.com/coorasse/Airesis/blob/develop/app/views/layouts/_head.html.erb#L87